dice.js

```diff
@@ -0,0 +1,47 @@
+const rollDice = function (numbRolls) {
```

Comment:
you can use a default parameter when numbRolls is not given ```const rollDice = function (numbRolls = 2)```, this will set the value of `numbRolls` to  `2` when no argument is given to the function.